codesearch/server/server.go

```diff
@@ -70,12 +71,17 @@ type codesearchServer struct {
 	scratchDirectory string
 }
 
-func makeDoc(name, repo, commitSha string, buf []byte) (types.Document, error) {
+func makeDoc(name, repoURLString, commitSha string, buf []byte) (types.Document, error) {
```

Comment:
the changes in this file are just manual fix-ups to accommodate the new proto format.